app/main.py

```diff
@@ -1 +1,52 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+    def __init__(self,
```

Comment:
Pass the paramters indents like in [this](https://stackoverflow.com/a/41591509) example